recipes/r-admm/meta.yaml

```
@@ -0,0 +1,102 @@
+{% set version = '0.3.3' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-admm
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/ADMM_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/ADMM/ADMM_{{ version }}.tar.gz
+  sha256: 1618e822808c68ec76f9060dc39c6ac2cc6f5dff0603a0359f727490bf8d7111
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - $RPATH/R.dll
+    - $RPATH/Rlapack.dll
+    - $RPATH/Rblas.dll
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - libopenblas
+    - _openmp_mutex   # [linux]
+    - llvm-openmp  # [osx]
+    - m2w64-libwinpthread-git  # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-matrix
+    - r-rcpp
+    - r-rcpparmadillo
+    - r-rdpack
+    - r-doparallel
+    - r-foreach
+    - libopenblas
```

Instead, keep BLAS implementation open, and put OpenMP dependency here:

```suggestion
    - libblas
    - libgomp          # [linux]
    - llvm-openmp      # [osx]
```